noxfile.py

```diff
@@ -200,30 +203,64 @@ def bundle(session):
 
 @nox.session(python=False)
 def sign(session):
-    """Sign the bundled distribution (OSX only)."""
+    """Sign the bundled distribution (macOS and Windows only)."""
     system = platform.system().lower()
 
-    if system != 'darwin':
-        session.skip('signing process is for OSX only')
-
-    session.run('security', 'find-identity', external=True)
-    session.run(
-        'codesign',
-        '--deep',
-        '--force',
-        '--verbose',
-        '--timestamp',
-        '--identifier',
-        OSX_BUNDLE_IDENTIFIER,
-        '--entitlements',
-        OSX_BUNDLE_ENTITLEMENTS,
-        '--options',
-        'runtime',
-        *session.posargs,
-        'dist/b2',
-        external=True
-    )
-    session.run('codesign', '--verify', '--verbose', 'dist/b2', external=True)
+    if system == 'darwin':
```

Comment:
please refactor into sign_darwin, sign_windows

Reply:
How would it be better? It would be less consistent, for sure, as we don't have `bundle_darwin`, etc. and the binary can not be cross-signed anyway as in such case having per-OS command could make sense.

Reply:
I mean instead of an enormous function with everything in it split signing for different operating systems into separate functions and then call them here

Reply:
Right ;)